index.bs

```diff
@@ -6996,7 +6996,7 @@ an |ownership type|, a |serialization internal map|, a |realm| and a |session|:
 
     <dt>|value| has a \[[DateValue]] [=internal slot=].
     <dd>
-    1. Set |serialized| to [=Call=]([=Date.toISOString=], |value|).
+    1. Set |serialized| to [=Call=]([=Date.toISOString=], «|value|»).
```

Comment:
Good catch, but doesn't it also require the `V` (`this`) argument?

```suggestion
    1. Set |serialized| to [=Call=]([=Date.toISOString=], null, «|value|»).
```

Reply:
Oh, yes, of course. In fact, that's the intended purpose of `|value|` as written without this patch, as `toISOString` method is an instance method for `Date` objects rather than a static method on the constructor. In other words, the spec text is correct.

That said, it looks like we were both confused by this proposal's use of `Date.toISOString` to refer to `Date.prototype.ISOString`. Rather than close this pull request outright, I've repurposed it to suggest renaming the reference to match how it is defined in ECMA262. If that seems desirable, then I'll be happy to rebase the branch in order to correct the commit message.

Reply:
Ah good point! And indeed fixing the reference to correctly point to the prototype chain makes that work.